analysis/include/BlFitHistos.h

```diff
@@ -22,98 +22,36 @@
 #include "TF1.h"
 #include <fstream>
 
-/**
```

Comment:
This file is still losing the Doxygen comments Sarah added. Others are too, need to figure out how to update this PR so it doesn't delete her work.